### ISSUE TITLE
Request for Test Set Color Labels

### ISSUE THREAD

**Issue by fei88kk** [morrisfl/stanford_cars_refined#1]
Dear Author,

Thank you for providing such a comprehensive open-source library, especially the color labels for the training set, which have been incredibly helpful for our experiments. As we continue with our testing, we are in need of the color labels for the test set as well. Would it be possible for you to kindly provide those? We greatly appreciate your assistance!!!!!
